shared/javascripts/tooltip.js

```diff
@@ -204,7 +204,7 @@ var privlyTooltip = {
           // Fill only the first three columns with the coresponding values from glyphArray[]
           // The rest of two columns are simetrical to the first two
           if(j <= 2) {
-            if(glyphArray[i * 3 + j] == "true") {
+            if(glyphArray[i * 3 + j] === "true") {
```

Comment:
I am not too sure about this. @smcgregor will be able to comment better. It depends on the type of values the glyphArray stores.